daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ledgerinteraction/grpcLedgerClient/AdminLedgerClient.scala

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Temporary stand-in for the real admin api clients defined in canton. Needed only for upgrades testing
```

Comment:
This would not solve the need for the AdminLedgerClient, as you can still only _vet_ and _unvet_ using the admin client.
If this was implemented, it would allow us to revert the bootstrap changes in the canton runner, so I'll add this TODO there.